libs/@guardian/react-crossword/src/components/Crossword.tsx

```diff
@@ -40,7 +40,7 @@ export const Crossword = ({ theme: userTheme, ...props }: CrosswordProps) => {
 	});
 
 	const moveFocus = useCallback(
-		(delta: { x: number; y: number }) => {
+		(delta: { x: number; y: number }, typing = false) => {
```

Comment:
if you make this an object, it helps describe what that `true` is doing further down:

`moveFocus({ x: -1, y: 0 }, true);` → `moveFocus({ x: -1, y: 0 }, { typing: true });`

```suggestion
		(delta: { x: number; y: number }, { typing = false }: { typing: boolean }) => {
```

Reply:
Thanks, I changed it to a single object (delta: {}, isTyping = false: {delta: {}; isTyping: boolean})